client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx

```diff
@@ -120,11 +120,11 @@ class CustomNameserversForm extends React.PureComponent {
 							onClick={ this.handleSubmit }
 							disabled={ this.props.submitDisabled }
 						>
-							{ translate( 'Save Custom Name Servers' ) }
+							{ translate( 'Save custom name servers' ) }
 						</FormButton>
 
 						<FormButton type="button" isPrimary={ false } onClick={ this.handleReset }>
-							{ translate( 'Reset to Defaults' ) }
+							{ translate( 'Reset to defaults' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Settings reset to defaults.' )` *ES Score: 8*